sdks/java/extensions/zetasketch/build.gradle

```diff
@@ -32,6 +33,7 @@ dependencies {
     compile library.java.slf4j_api
     compile library.java.vendored_guava_26_0_jre
     compile project(path: ":sdks:java:core", configuration: "shadow")
+    compile "com.google.auto.value:auto-value-annotations:1.6.3"
```

Comment:
We already include auto-value-annotations in BeamModulePlugin, why is it needed here?

Reply:
See https://github.com/apache/beam/pull/13686#discussion_r574955593

Reply:
Let's use the library you created.

Reply:
```suggestion
```

Reply:
Oops - my bad on that.